### PR TITLE
[FEAT]: Adding carousel in hero section

### PR DESCRIPTION
**Pull request by Harshitazz**
fixes issue: #9
Screenshot 2024-05-11 160522
carousel is being added.

### CODE REVIEW

File: frontend/src/components/Hero.jsx [hunk 1]
```diff
@@ -1,46 +1,72 @@
...
-                src="https://images.apollo247.in/images/ic-doctor.svg"
-                alt="icon"
-                className="hidden mr-2 md:mr-4 size-8"
-              />
-              Book an Appointment <IoIosArrowForward className="ml-2" />
-            </button>
+  return (
+    <Slider {...settings}>
+      {images.map((image, index) => (
+        <div key={index} className={" w-full  bg-black/20  "}>
```
Comment (itsmohit097) [inline frontend/src/components/Hero.jsx:31]: Remove the classes form here it is not working here properly

File: frontend/src/components/Hero.jsx [hunk 2]
```diff
@@ -1,46 +1,72 @@
...
+          <div className="w-full h-[100vh] relative md:bg-left-top bg-cover bg-no-repeat bg-center" style={{ backgroundImage: `url(${image})` }}>
```
Comment (itsmohit097) [inline frontend/src/components/Hero.jsx:32]: Below this wrap all code under a new div with the same classes that I told you to remove above

**Review by itsmohit097 — CHANGES_REQUESTED** [review]

**Comment by itsmohit097** [comment]
> Screenshot 2024-05-11 160522 carousel is being added.

Make the suggested changes and less the speed of autoplay 
After that i will merge the pull request

**Comment by Harshitazz** [comment]
done

**Pull request merged**